### ISSUE TITLE
Email: IBM Comments on WCAG 3.0 Draft (8 - Plain language summaries)

### ISSUE THREAD

**Issue by cwadamsoforacle** [w3c/wcag3#262]
[Comment from email:](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Mar/0005.html)
Plain language summaries
We support the inclusion of plain language summaries and think these are helpful short descriptions of the topics covered in WCAG 3. We do have a few concerns, questions, and suggested improvements for plain language summaries.
•	Including the Plain Language Summary at the beginning of each section, interspersed, and in-between other normative and informative content is a bit confusing.  If the primary intent is to serve a specific audience, perhaps a consolidated view of only Plain Language content should be made available using a “plain language’ filter. For example, the opposite of what is shown now with all the Plain Language sections collapsed and the complex language shown/expanded by default. 
•	Introduction: It is a little confusing that the plain language summary is titled "Introduction" yet the actual introduction section has no text other than, “This section is non-normative.” It instead summarizes the content in the sub sections below, none of which have "Introduction" in their title. Perhaps it should include an explanation that it summarizes sub-sections 1.1-1.4. The plain language summary for the structure sections is clearer since the bulleted list with one bullet for each sub-section that contains the distinctive word from the headings of the sub-section AND there is text with the diagram that gives a good introduction to the topic as well.
•	In an effort to keep the plain language sections, a lot of sub-section details aren't present. Wonder if this may cause some confusion or missed information. In other words, the plain language summary cannot be relied upon on its own to understand the full picture. An example is Section 2. Structure of these guidelines where the topics of critical errors, outcome ratings and the structure of methods is not included. 
•	The Figure 1 core structure diagram could potentially be combined with the plain language summary for better comprehension. Some people understand pictures better, and some understand plain language better, and the combination may be better yet. 
•	Does each plain language summary need to have the last line announcing the "End of summary..."? Seems that the call-out box styling is sufficient to indicate that, especially if it is also defined with a landmark region role for AT programmatic identification of the block of information.
[IBM comments on the WCAG 3 FPWD -March 2.docx](https://github.com/w3c/silver/files/6174052/IBM.comments.on.the.WCAG.3.FPWD.-March.2.docx)
